lib/heroku/kensa/check.rb

```diff
@@ -230,7 +234,7 @@ def call!
           end
 
           check "syslog_drain_url is returned if required" do
-            return true unless data.has_key?("requires") && data["requires"].include?("syslog_drain")
+            return true unless api_requires?("syslog_drain")
 
             drain_url = response['syslog_drain_url']
```

Comment:
To be strictly correct, they may actually return `log_drain_url` instead, because [log drains don't have to be `syslog://`-based](https://devcenter.heroku.com/articles/accessing-app-logs#add-on-manifest).